gob/dcos/marathon/linkerd.json

```diff
@@ -0,0 +1,42 @@
+{
+  "id": "linkerd",
+  "instances": 3,
```

Comment:
consider making note of this value in the readme, it will vary depending on cluster size. it should equal the total number of public and private nodes.